### DISCUSSION TITLE
Carousel : html content/image slider (with mouse control)

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#867] [Q&A] [answered]
Hi 

Is there an example, or does someone experienced a carousel with mouse control?

I saw the excellent [carousel](https://codepen.io/dmitrysharabin/pen/jjKwGG) demo from @DmitrySharabin , but I can't slide content with mouse :(

I tried to use [Tiny-slider](https://ganlanyuan.github.io/tiny-slider/demo/#arrowKeys_wrapper) in my [codepen](https://codepen.io/galinhalx/pen/YzebzOa), but When I'm trying to add a content (image or html content), images are displayed one under the other, and nothing slides

Is this simply incompatible with mavo (because it uses javascript)? or is there another mouse carousel library compatible with mavo that someone knows about?

**Comment by GalinhaLX** [comment, marked as answer]
found this [library](https://splidejs.com/) which works fine with mavo

**Reply by GalinhaLX** [reply]
well almost fine :- I'm able to edit a slide with html content or image. And almost able to add a slide (It's there, but I can't access it to edit it). Needs javascript skills to both use mavo and javascript from library maybe

**Comment by joyously** [comment]
Carousels are notoriously bad for user experience and SEO. 
I suggest that Mavo does not deliberately support that.

**Reply by GalinhaLX** [reply]
I agree, except that new v4 of this [carousel](https://splidejs.com/guides/version4/) covers it 

more information about it accessibility + tests [here](https://splidejs.com/guides/accessibility/)

**Comment by karger** [comment]
if implement the way I'd expect in mavo, which would involve visually 
styling some fixed content, i doubt it would affect SEO, but I agree 
it's not a great UX.

**Comment by LeaVerou** [comment]
I'd suggest finding a carousel web component. Because web components are designed to be used with HTML syntax only, they'd work fine with Mavo.
